classes/village/VillageBuildingDto.php

```diff
@@ -13,16 +13,17 @@ public function __construct(
         public ?int $construction_progress = null,
         public ?int $construction_progress_required = null,
         public ?int $construction_progress_last_updated = null,
-        // display-only properties
-        public string $name = "",
-        public string $description = "",
-        public string $phrase = "",
-        public ?int $materials_construction_cost = null,
-        public ?int $food_construction_cost = null,
-        public ?int $wealth_construction_cost = null,
-        public ?int $construction_time = null,
-        public ?string $construction_time_remaining = null,
-        public ?int $requirements_met = null,
+        public bool $construction_boosted = false,
+        public string $construction_time_remaining = '',
```

Comment:
What's the expected value of this field? Should we use int or a date type instead?

Reply:
`$construction_time_remaining` is a formatted display string.